pyocd/debug/semihost.py

```diff
@@ -479,32 +479,32 @@ def _get_args(self, args, count):
         else:
             return args
 
-    def _get_string(self, ptr, length=None):
+    def _get_data(self, ptr, length=None):
```

Comment:
Hmm, maybe we should just separate `._get_string()` from `._get_data()`? The only difference being null-termination support; return type would be the same. Not necessary, just thinking.

And probably should remove the leading underscore since it's not actually a private method and is called from outside the class. (My bad!)